test/stressHistoFit.cxx

```diff
@@ -1129,7 +1129,8 @@ void init_structures()
 //    noGraphAlgos.push_back( algoType( "Minuit",      "Migrad",      "GQE0", CompareResult()) );
     noGraphAlgos.push_back( algoType( "Minuit",      "Migrad",      "GQL0", CompareResult()) );
 
-   noGraphErrorAlgos.push_back( algoType( "Fumili",      "Fumili",      "Q0", CompareResult()) );
+    noGraphErrorAlgos.push_back(algoType("Minuit2", "Fumili", "Q0", CompareResult()));
+    //noGraphErrorAlgos.push_back(algoType("Fumili", "Fumili", "Q0", CompareResult()));
```

Comment:
```suggestion
```
I guess since "Fumili" is part of "Minuit2", the combination "Fumili" "Fumili" is not intended to work anyway, right? In this case, it could be better to remove the commented-out line, because it suggests that it should work in principle, but it just doesn't work right now.

Reply:
Fumili/Fumili exists , it is an old implementation translated directly from Fortran and available in the TFumili class. However, I would like to deprecate this as a legacy implementation and have the recommended one in Minuit2. 
Now I have commented out this test, because it did not work when computing the derivatives internally when using TFumili. This was not working before too, so I have for the time being decided to remove this test.

Reply:
Okay, thank you for the clarification!